<!DOCTYPE html>
<html>

<style>
seq {font-family:Monaco, "Courier New";
  font-size:0.9em;
  text-indent: 2.7em;
}

align.small {
    font-family:Monaco, "Courier New";
    font-size:0.9em;
    line-height:65%;
}

align.long {
  font-family:Monaco, "Courier New";
  font-size:0.8em;
  line-height:70%;
}

s5 {color:#6baed6;}
s7 {color:#fc9272;}
p5 {color:#08519c;}
p7 {color:#a50f15;}
me {color:#969696;}
tso {color:#2ca25f;}
cbc {color:#f768a1;}
umi {color:#807dba;}
t7 {color:blue;}
pe1 {color:#bcbddc;}
pe2 {color:#9ebcda;}
w1 {color:#f03b20;}
h3 {font-family:verdana;}

</style>

<head>
<title>BD Rhapsody WTA</title>
</head>
<body>

<h1><a href="https://www.nature.com/articles/s41592-018-0255-0" target="_blank">BD Rhapsody WTA</a></h1>

<p><span style="font-family:verdana; font-size:1.1em;">BD Rhapsody WTA is a nanowell-based commercial system like <a href="https://teichlab.github.io/scg_lib_structs/methods_html/Microwell-seq.html" target="_blank">Microwell-seq</a>. They use similar split-pool approach to generate their oligos on magnetic beads. The first publication is in <i>Nature Methods</i> 16, 75â€“78 (2019). Note: I don't have the full sequence details from each step of the protocol. Sequences presented here are based on educational guess from the sequencing data. The final libary structure should be accurate though. Click <a href="../data/GMX_BD-Rhapsody-Single-Cell-Analysis-System-Instrument_UG_EN.pdf" target="_blank">here</a> for the guide to use the machine, and click <a href="../data/GMX_BD-Rhapsody-WTA-alpha-Protocol_UG_EN.pdf" target="_blank">here</a> to see the off-machine protocol.</span></p>

<br></br>

<h2>Adapter and primer sequences:</h2>
<h3>Sequence used during the experiment:</h3>
<seq>
<p>Indexed-Bead-oligo: |--5'-<s5>GACGCTCTTCCGATCT</s5><cbc>[CLS1]</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>[CLS2]</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(T)<sub>18</sub> -3'</p>
<p>-- Cells are determined as different combination of <cbc>[CLS1]</cbc>, <cbc>[CLS2]</cbc> and <cbc>[CLS3]</cbc>, each is 9-bp long.</p>
<p>-- There are 97 different sequences each, so you have a total of 97x97x97 = 912,673 different combinations.</p>
<p>-- Clike here to see the sequences of <a href="../data/BD_CLS1.txt", target="_blank">CLS1</a>, <a href="../data/BD_CLS2.txt", target="_blank">CLS2</a>, and <a href="../data/BD_CLS3.txt", target="_blank">CLS3</a>.</p>
<p>Randomer in the kit: 5'- <s7>TCAGACGTGTGCTCTTCCGATCT</s7>NNNNNNNNN -3'</p>
<p>Pre-amp forward primer: 5'- <s5>GACGCTCTTCCGATCT</s5> -3'</p>
<p>Pre-amp reverse primer: 5'- <s7>TCAGACGTGTGCTCTT</s7> -3'</p>
<p>Library Forward primer: 5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5><s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5> -3'</p>
<p>Library Reverse primer: 5'- <p7>CAAGCAGAAGACGGCATACGAGAT</p7>[8-bp sample index]<s7>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT</s7> -3'</p>
<p>Illumina Truseq Read 1 primer: 5'- <s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5> -3'</p>
<p>Illumina Truseq Read 2 primer: 5'- <s7>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT</s7> -3'</p>
<p>Sample index sequencing primer: 5'- <s7>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</s7> -3'</p>
<p>Illumina P5 adapter: 5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5> -3'</p>
<p>Illumina P7 adapter: 5'- <p7>CAAGCAGAAGACGGCATACGAGAT</p7> -3'</p>
</seq>

<br></br>

<h2>Step-by-step library generation</h2>
<h3>(1) Capture cells, load indexed beads, lyse cells, anneal mRNA to indexed-bead-oligo, collect and pool beads into a 1.5 ml Eppendorf tube, and reverse transcription in the tube:</h3>
<pre>
<align class="small">
|--5'-<s5>GACGCTCTTCCGATCT</s5><cbc>[CLS1]</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>[CLS2]</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(T)<sub>18</sub>---------->
                                                                           (A)<sub>n</sub>XXXXXXXXX...XXXXXXXXX -5'
</align>
</pre>

<h3>(2) Heat the beads to get rid of RNA, only keep the cDNA on the beads, then perform second strand synthesis using random priming (the Randomer in the kit) with Klenow at 37 degree. This step is called random priming and extension (RPE) in their off-machine protocol:</h3>
<pre>
<align class="small">
|--5'-<s5>GACGCTCTTCCGATCT</s5><cbc>[CLS1]</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>[CLS2]</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(dT)XXXXXXXXXXXX...XXXXXXXXXXXX -3'
                                                                                       <--------NNNNNNNNN
                                                                                                         <s7>TCTAGCCTTCTCGTGTGCAGACT</s7> -5'
</align>
</pre>

<h3>(3) This is the product after the RPE step:</h3>
<pre>
<align class="small">
3'- <s5>CTGCGAGAAGGCTAGA</s5><cbc>[CLS1]</cbc><pe1>TGACCGGACGCT</pe1><cbc>[CLS2]</cbc><pe2>CCATCGCCACTGT</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(pA)XXXXXXXXXXXX...XXXXXXXXXXXX<s7>TCTAGCCTTCTCGTGTGCAGACT</s7> -5'
</align>
</pre>

<h3>(4) Perform RPE PCR using Universal Oligo (I assume this is a mix of Pre-amp forward primer and Pre-amp reverse primer):</h3>
<pre>
<align class="small">
5'- <s5>GACGCTCTTCCGATCT</s5>---------->
3'- <s5>CTGCGAGAAGGCTAGA</s5><cbc>[CLS1]</cbc><pe1>TGACCGGACGCT</pe1><cbc>[CLS2]</cbc><pe2>CCATCGCCACTGT</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(pA)XXXXXXXXXXXX...XXXXXXXXXXXX<s7>TCTAGCCTTCTCGTGTGCAGACT</s7> -5'
                                                                                                    <----------<s7>TTCTCGTGTGCAGACT</s7> -5'
</align>
</pre>

<h3>(5) Purfify the RPE PCR product:</h3>
<pre>
<align class="small">
5'- <s5>GACGCTCTTCCGATCT</s5><cbc>[CLS1]</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>[CLS2]</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(dT)XXXXXXXXXXXX...XXXXXXXXXXXX<s7>AGATCGGAAGAGCACACGTCTGA</s7> -3'
3'- <s5>CTGCGAGAAGGCTAGA</s5><cbc>[CLS1]</cbc><pe1>TGACCGGACGCT</pe1><cbc>[CLS2]</cbc><pe2>CCATCGCCACTGT</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(pA)XXXXXXXXXXXX...XXXXXXXXXXXX<s7>TCTAGCCTTCTCGTGTGCAGACT</s7> -5'
</align>
</pre>

<h3>(6) Performing WTA Index PCR using Library Forward primer and Library Reverse primer:</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5><s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5>---------->
                                          5'- <s5>GACGCTCTTCCGATCT</s5><cbc>[CLS1]</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>[CLS2]</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(dT)XXX...XXX<s7>AGATCGGAAGAGCACACGTCTGA</s7> -3'
                                          3'- <s5>CTGCGAGAAGGCTAGA</s5><cbc>[CLS1]</cbc><pe1>TGACCGGACGCT</pe1><cbc>[CLS2]</cbc><pe2>CCATCGCCACTGT</pe2><cbc>[CLS3]</cbc><umi>[8-bp UMI]</umi>(pA)XXX...XXX<s7>TCTAGCCTTCTCGTGTGCAGACT</s7> -5'
                                                                                                                     <----------<s7>TCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG</s7>[8-bp sample index]<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(7) Final library structure:</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5><s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5><cbc>NNNNNNNNN</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>NNNNNNNNN</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>NNNNNNNNN</cbc><umi>NNNNNNNN</umi>(dT)XXX...XXX<s7>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</s7>NNNNNNNN<p7>ATCTCGTATGCCGTCTTCTGCTTG</p7> -3'
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5><s5>AGAAAGGGATGTGCTGCGAGAAGGCTAGA</s5><cbc>NNNNNNNNN</cbc><pe1>TGACCGGACGCT</pe1><cbc>NNNNNNNNN</cbc><pe2>CCATCGCCACTGT</pe2><cbc>NNNNNNNNN</cbc><umi>NNNNNNNN</umi>(pA)XXX...XXX<s7>TCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG</s7>NNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
            <p5>Illumina P5</p5>                 <s5>Truseq Read 1</s5>            <cbc>9bp</cbc>     <pe1>linker1</pe1>      <cbc>9bp</cbc>     <pe2>linker2</pe2>       <cbc>9bp</cbc>      <umi>8bp</umi>        cDNA             <s7>Truseq Read 2</s7>             8bp         <p7>Illumina P7</p7>
                                                                 <cbc>CLS1</cbc>                 <cbc>CLS2</cbc>                  <cbc>CLS3</cbc>     <umi>UMI</umi>                                               sample index
</align>
</pre>

<br></br>

<h2>Library sequencing:</h2>

<h3>(1) Add Truseq Read 1 primer to sequence the first read (at least 60 cycles, bottom strand as template, these are the cell barcodes and UMI. The combination of CLS1/2/3 is the cell barcode):</h3>
<pre>
<align class="long">
                             5'- <s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5>----------------------------------------------------------->
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5><s5>AGAAAGGGATGTGCTGCGAGAAGGCTAGA</s5><cbc>NNNNNNNNN</cbc><pe1>TGACCGGACGCT</pe1><cbc>NNNNNNNNN</cbc><pe2>CCATCGCCACTGT</pe2><cbc>NNNNNNNNN</cbc><umi>NNNNNNNN</umi>(pA)XXX...XXX<s7>TCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG</s7>NNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(2) Add Sample index sequencing primer to sequence sample index (8 cycles, bottom strand as template):</h3>
<pre>
<align class="long">
                                                                                                                                   5'- <s7>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</s7>------->
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5><s5>AGAAAGGGATGTGCTGCGAGAAGGCTAGA</s5><cbc>NNNNNNNNN</cbc><pe1>TGACCGGACGCT</pe1><cbc>NNNNNNNNN</cbc><pe2>CCATCGCCACTGT</pe2><cbc>NNNNNNNNN</cbc><umi>NNNNNNNN</umi>(pA)XXX...XXX<s7>TCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG</s7>NNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(3) Cluster regeneration, and add Illumina Truseq Read 2 primer to sequence read 2 (top strand as template, these are the cDNA reads):</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5><s5>TCTTTCCCTACACGACGCTCTTCCGATCT</s5><cbc>NNNNNNNNN</cbc><pe1>ACTGGCCTGCGA</pe1><cbc>NNNNNNNNN</cbc><pe2>GGTAGCGGTGACA</pe2><cbc>NNNNNNNNN</cbc><umi>NNNNNNNN</umi>(dT)XXX...XXX<s7>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</s7>NNNNNNNN<p7>ATCTCGTATGCCGTCTTCTGCTTG</p7> -3'
                                                                                                                               <-------<s7>TCTAGCCTTCTCGTGTGCAGACTTGAGGTCAGTG</s7> -5'
</align>
</pre>

</body>
</html>
